pycbc/waveform/nr_waveform.py

```diff
@@ -181,10 +179,11 @@ def get_param(value):
             curr_h_real = curr_amp * numpy.cos(curr_phase)
             curr_h_imag = curr_amp * numpy.sin(curr_phase)
             curr_ylm = lal.SpinWeightedSphericalHarmonic(theta, phi, -2, l, m)
-            hp += curr_h_real * curr_ylm.real - curr_h_imag * curr_ylm.imag
+            print curr_ylm
+            hp += curr_h_real * curr_ylm.real + curr_h_imag * curr_ylm.imag
             # FIXME: No idea whether these should be minus or plus, guessing
             #        minus for now. Only affects some polarization phase
-            hc += - curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
+            hc += + curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
```

Comment:
Hi Ian,

I redid the maths and get a different convention as I wrote in an email a
few days ago.
Can you redo your checks leaving the signs as the were but change
curr_h_imag = curr_amp * numpy.sin(curr_phase)
to
curr_h_imag = - curr_amp * numpy.sin(curr_phase)   #Minus sign here on the
RHS
before pushing the change to master?

Many thanks,
Patricia.

On 29 October 2015 at 01:45, spxiwh <notifications@github.com> wrote:

> In pycbc/waveform/nr_waveform.py
> <https://github.com/spxiwh/pycbc/pull/1#discussion_r43360978>:
>
> >              # FIXME: No idea whether these should be minus or plus, guessing
> >              #        minus for now. Only affects some polarization phase
> > -            hc += - curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real
> > +            hc += + curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real
>
> Hi Patricia, can you check out the branch testing_phase_conventions, make
> the change you suggest, and then push that change to
> testing_phase_conventions. I want to merge this soon, but also want to
> agree on this before merging.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spxiwh/pycbc/pull/1/files#r43360978>.
>



-- 
Patricia Schmidt
Postdoctoral Scholar in Physics

California Institute of Technology
TAPIR, 350-17
1200 E. California Boulevard
Pasadena, California 91125
USA

email: pschmidt@tapir.caltech.edu <pschmidt@ligo.caltech.edu>